courses/info.yml

```diff
@@ -0,0 +1,2 @@
+- pyladies
+- mi-pyt
```

Comment:
Could you put this under an `order:` key, so we can put more stuff in `info.yml` later, and so that it's obvious what this means?